Riot/Managers/Theme/Themes/DefaultTheme.swift

```diff
@@ -105,6 +105,14 @@ class DefaultTheme: NSObject, Theme {
     
     var roomCellOutgoingBubbleBackgroundColor: UIColor = UIColor(rgb: 0xE7F8F3)
     
+    var roomCellLocalisationTextColor: UIColor = UIColor(rgb: 0x17191C)
```

Comment:
This color seems redundant with `primaryContent`

Reply:
The problem is, "primaryContent" color is White on dark theme, so it didn't work on the white background of the cell.